doc/man3/X509_cmp_time.pod

```diff
@@ -43,12 +53,24 @@ X509_cmp_current_time() return 0 on error.
 =head1 RETURN VALUES
 
 X509_cmp_time() and X509_cmp_current_time() return -1 if B<asn1_time>
-is earlier than, or equal to, B<cmp_time> (resp. current time), and 1
+is earlier than, or equal to, B<in_tm> (resp. current time), and 1
 otherwise. These methods return 0 on error.
 
+X509_cmp_timeframe() returns 0 if B<vpm> is not NULL and the verification
+parameters do not contain B<X509_V_FLAG_USE_CHECK_TIME>
+but do contain B<X509_V_FLAG_NO_CHECK_TIME>. Otherwise it returns
+1 if the end time is not NULL and the reference time (which has determined as
+stated above) is past the end time, -1 if the start time is not NULL and the
+reference time is before, else 0 to indicate that the reference time is in range
+(implying that the end time is not before the start time if both are present).
```

Comment:
What about changing _implying_ to _on the assumption that_.  Implying kind of means that what follows is true and it needn't be.

Reply:
I really meant 'implying' in the propositional/logical sense: 
If the result is 0 then ref time <= end and and start < ref time, which does imply start < end. So the correct range is a consequence here.
Conversely, if (as usual) start < end and the ref time is in between, then the result is 0. So the correct range can also be seen as an assumption.